### PR TITLE
windows: Add window-properties.ui

### PR DESCRIPTION
**Pull request by rbuj**

### CODE REVIEW

File: capplets/windows/window-properties.ui [hunk 1]
```diff
@@ -0,0 +1,684 @@
...
+          <object class="GtkButtonBox">
+            <property name="can_focus">False</property>
+            <property name="layout_style">end</property>
+            <child>
+              <object class="GtkButton" id="button1">
+                <property name="label">gtk-help</property>
+                <property name="visible">True</property>
+                <property name="can_focus">True</property>
+                <property name="receives_default">True</property>
+                <property name="use_stock">True</property>
```
Comment (sc0w) [inline capplets/windows/window-properties.ui]: `use-stock` is deprecated

https://developer.gnome.org/gtk3/stable/GtkButton.html#GtkButton--use-stock

you can use icon name "help-browser" instead

File: capplets/windows/window-properties.ui [hunk 2]
```diff
@@ -0,0 +1,684 @@
...
+                <property name="position">0</property>
+              </packing>
+            </child>
+            <child>
+              <object class="GtkButton" id="button2">
+                <property name="label">gtk-close</property>
+                <property name="visible">True</property>
+                <property name="can_focus">True</property>
+                <property name="receives_default">True</property>
+                <property name="use_stock">True</property>
```
Comment (sc0w) [inline capplets/windows/window-properties.ui]: `use-stock` is deprecated

https://developer.gnome.org/gtk3/stable/GtkButton.html#GtkButton--use-stock

you can use icon name "window-close" instead

File: capplets/windows/window-properties.ui [hunk 3]
```diff
@@ -0,0 +1,684 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Generated with glade 3.22.1 -->
+<interface>
+  <requires lib="gtk+" version="3.20"/>
```
Comment (sc0w) [inline capplets/windows/window-properties.ui]: `mate-control-center` requires gtk 3.22

Comment (rbuj) [reply capplets/windows/window-properties.ui]: @sc0w What version of glade do you use?

Captura de pantalla a 2019-12-22 14-07-49

Comment (sc0w) [reply capplets/windows/window-properties.ui]: I didn't open the file with glade, I just reviewed the code

Glade have this issue, already fixed, reported by me:
https://gitlab.gnome.org/GNOME/glade/issues/289

Comment (rbuj) [reply capplets/windows/window-properties.ui]: OK, the UI file was verified with https://github.com/GNOME/glade/commit/213447c582623555f7bda571a2d9132087e27979.

**Pull request merged**